neuro_san/registries/bing_search.hocon

```diff
@@ -65,20 +72,20 @@
             #
             # Langchain Tools
             #
-            #   1️)  "bing_search" - Uses Bing to perform web searches.
-            #   2️)  "tavily_search" - Uses Tavily for search queries.
-            #   3️)  "requests_get" - Makes an HTTP GET request.
-            #   4️)  "requests_post" - Makes an HTTP POST request.
-            #   5️)  "requests_patch" - Makes an HTTP PATCH request.
-            #   6️)  "requests_put" - Makes an HTTP PUT request.
-            #   7️)  "requests_delete" - Makes an HTTP DELETE request.
-            #   8)  "requests_toolkit" - Contains all of the request tools.
+            #   * "bing_search" - Uses Bing to perform web searches.
+            #   * "tavily_search" - Uses Tavily for search queries.
+            #   * "requests_get" - Makes an HTTP GET request.
+            #   * "requests_post" - Makes an HTTP POST request.
+            #   * "requests_patch" - Makes an HTTP PATCH request.
+            #   * "requests_put" - Makes an HTTP PUT request.
+            #   * "requests_delete" - Makes an HTTP DELETE request.
+            #   * "requests_toolkit" - Contains all of the request tools.
```

Comment:
Get rid of the numbering.
First a weird subscript was being used in the file.
Second, any time anything was added all the entries would have to change.